<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Change saving path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionOpen_video"/>
    <addaction name="actionOpen_folder"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QMenu" name="menuDevice">
     <property name="title">
      <string>Device</string>
     </property>
     <addaction name="actionCPU"/>
     <addaction name="actionGPU"/>
    </widget>
    <widget class="QMenu" name="menuModel">
     <property name="title">
      <string>Model</string>
     </property>
     <addaction name="actionyolov5s"/>
     <addaction name="actionyolov5m"/>
     <addaction name="actionyolov5l"/>
     <addaction name="actionyolov5x"/>
    </widget>
    <addaction name="menuModel"/>
    <addaction name="menuDevice"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTerminal"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuSetting"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_image">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCPU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CPU</string>
   </property>
  </action>
  <action name="actionGPU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GPU</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="text">
    <string>Open folder</string>
   </property>
  </action>
  <action name="actionOpen_video">
   <property name="text">
    <string>Open video</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionyolov5s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov5s</string>
   </property>
  </action>
  <action name="actionyolov5m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov5m</string>
   </property>
  </action>
  <action name="actionyolov5l">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov5l</string>
   </property>
  </action>
  <action name="actionyolov5x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov5x</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
